apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -811,7 +811,7 @@ const CTA = ({ data }: { data: GetByViewerResponse }) => {
 const Actions = () => {
   return (
     <div className="hidden items-center md:flex">
-      <TeamsFilter useProfileFilter popoverTriggerClassNames="mb-0" showVerticalDivider={true} />
+      <TeamsFilter popoverTriggerClassNames="mb-0" showVerticalDivider={true} />
```

Comment:
It was a way to opt-in to profile based quering. So, disabling that for event-types now.